app-emulation/docker/docker-1.11.2-r1.ebuild

```diff
@@ -0,0 +1 @@
+docker-9999.ebuild
```

Comment:
Oh, I see. It was added because you are changing `docker.service`. You will need to bump the docker version (I think it's 1.12.1 at this point) to pull in the changes.